environment_agency_flood/env_agency_flood.py

```diff
@@ -360,7 +360,7 @@ def get_args() -> argparse.Namespace:
     """Command-line arguments"""
 
     parser = argparse.ArgumentParser(description=DESCRIPTION)
-    parser.add_argument('-d', '--date', required=True, type=lambda s: datetime.datetime.strptime(s, '%Y-%m-%d'), help="ISO UTC date")
+    parser.add_argument('-d', '--date', required=True, type=datetime.date.fromisoformat, help="ISO UTC date")
```

Comment:
Python 3.5 datetime doesn't have date. Which version is on the server?

Reply:
I don't know how Python is set up on the server. There's no reason we can't use virtual environments for each pipeline. This means we should be able to use a recent Python version such as 3.6 or newer.

Reply:
The basic installed version on the operating system is 2.7 and 3.5. I don't know whether or not a virtual environment is used to run the actual pipelines.

```
sa_cs1jsth@ufacqt:~$ python --version
Python 2.7.12

sa_cs1jsth@ufacqt:~$ python3 --version
Python 3.5.2
```